server/ec2spotmanager/templates/config/index.html

```diff
@@ -6,8 +6,14 @@
 <div class="panel panel-default">
 	<div class="panel-heading"><i class="glyphicon glyphicon-scale"></i> Available Pool Configurations</div>
 	<div class="panel-body">
-		<div class="btn-group">
-			<a href="{% url 'ec2spotmanager:configcreate' %}" class="btn btn-default">Create Configuration</a>
+		<div class="dropdown">
```

Comment:
This looks like it will work, but it doesn't match the model we talked about for supporting multiple providers from a single config. I would just revert all of the template and view changes, but with the field renames as required.